ql/ql/src/qlpack.yml

```
@@ -5,3 +5,5 @@ dbscheme: ql.dbscheme
 suites: codeql-suites
 defaultSuiteFile: codeql-suites/ql-code-scanning.qls
 extractor: ql
+dependencies:
+  codeql/typos: "*"
```

For now, can you be explicit about the dependency here? This will require manually bumping when you change versions.

```suggestion
  codeql/typos: 0.0.1
```

We are working on something to make this simpler. See https://github.com/github/codeql-core/issues/2741